app/scripts/templates/permissions.mustache

```diff
@@ -0,0 +1,32 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-permissions-header">{{#t}}%(serviceName)s would like to know…{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success"></div>
+
+    <div id="permissions">
+      <p>
+        <span>{{#t}}Email{{/t}}</span>
+        <span class="email">{{ email }}</span>
+      </p>
+    </div>
+
+    <div class="privacy-links">
+      {{#t}}By proceeding, I agree to the <a id="service-tos" href="%(termsUri)s">Terms of Service</a> and <a id="service-pp" href="%(privacyUri)s">Privacy Notice</a> of %(serviceName)s (%(serviceUri)s).{{/t}}
```

Comment:
Should `serviceUri` always be on its own line? If so, should a `<br />` or `<p>` be added?